mlflow/R/mlflow/tests/testthat/test-tracking-runs.R

```diff
@@ -316,19 +316,22 @@ test_that("mlflow_log_batch() works", {
 
   expect_setequal(
     metrics$key,
-    c("mse", "rmse")
+    c("mse", "rmse", "nan", "Inf", "-Inf")
   )
   expect_setequal(
-    metrics$value,
+    metrics$value[metrics$name %in% c("mse", "rmse")],
     c(23, 42)
   )
+  expect_true(all(is.nan(metrics$value[metrics$key == "nan"])))
+  expect_true(all(1.7976931348623157e308 <= (metrics$value[metrics$key == "Inf"])))
```

Comment:
Could we not just check that the values are inf & -inf here?

Reply:
Ah, we can not unfortunately since this varies for different stores. 
E.g. SQL stores will return Double.MAX but file store will return Inf.